packages/cozy-intent/src/api/services/NativeMessenger.ts

```diff
@@ -8,17 +10,29 @@ import {
   strings
 } from '../../api'
 
+export const logger = Minilog('cozy-intent')
+
 const log = debug('NativeMessenger')
 
 export class NativeMessenger implements Messenger {
   private injectJavaScript?: (data: string) => void
   private listener?: MessageListener
+  private webviewUri?: string
 
   constructor(webviewRef: WebviewRef) {
     this.injectJavaScript = webviewRef.injectJavaScript
+    this.webviewUri =
+      webviewRef.props.source.uri ?? webviewRef.props.source.baseUrl
   }
 
   public postMessage = (message: Record<string, unknown>): void => {
+    if (message.error) {
```

Comment:
Personally I think the error wrap should be closer to the actual method call. But if this implementation suits your purpose that's fine for me